README.md

```diff
@@ -293,7 +294,9 @@ Connection plugin manager initializes, triggers, and cleans up a chain of connec
 The figure above shows a simplified workflow of the connection plugin manager.  
 Starting at the top, when a JDBC method is executed by the driver, it is passed to the connection plugin manager. From the connection plugin manager, the JDBC method is passed in order of plugins loaded like a chain. In this figure, `Custom Plugin A` to `Custom Plugin B` and finally to `Default Plugin` which executes the JDBC method and returns the result back up the chain.
 
-By default, Enhanced Failure Monitoring is loaded. Additional custom plugins can be implemented and used alongside existing ones. Plugins can be chained together in a desired order. Loading custom plugins will not include Enhanced Failure Monitoring unless explicitly stated through the `connectionPluginFactories` parameter.
+The failover functionality is implemented by the Failover Connection Plugin. By default, Enhanced Failure Monitoring and the Failover Connection Plugin are loaded.
```

Comment:
Don't think this adds anything of value.

Reply:
Or at least should be moved to the Failover section

Reply:
Reworded